magento/inventory
# Disable assign source for default stock (UI)

### Description
Disable "Assign Sources" on default stock in adminhtml.

This pull request handles the @aleron75 PR https://github.com/magento-engcom/msi/pull/317 for the adminhtml side

### Fixed issues
Fix adminhtml side of issue #313 

### Manual testing scenarios
1. Open default stock
2. "Assign Sources" button should be disabled
3. Open any other stock (or create e a new one)
4. "Assign Sources" button should be enabled


### Contribution checklist
